python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -1208,8 +1242,9 @@ def get_subset_selection_for_context(
     select: Optional[str],
     exclude: Optional[str],
     dagster_dbt_translator: DagsterDbtTranslator,
-) -> List[str]:
-    """Generate a dbt selection string to materialize the selected resources in a subsetted execution context.
+) -> Tuple[List[str], Optional[str]]:
```

Comment:
Since this can only be "empty", let's use `Final` instead of `str`

Reply:
seems like you can't use Final with function annotations unless I'm missing something https://mypy.readthedocs.io/en/stable/final_attrs.html#details-of-using-final